lithium/lithium.py

```diff
@@ -154,11 +154,9 @@ def processOptions():
     if args.chunksize:
         minimizeMin = args.chunksize
         minimizeMax = args.chunksize
-        repeat = "last"
```

Comment:
I think these lines are supposed to set the global variable 'minimizeRepeat', like the rough equivalents before commit 0e3dd3dea8476d3742266549646fa31210209fbb did. Also, was it intentional that the value was changed from "never" to "last" when chunksize is set?